install.sh

```diff
@@ -104,15 +153,18 @@ mkdir -p $STGDIR
 mkdir -p $STGDIR/scripts
 mkdir -p $STGDIR/repo
 mkdir -p /var/log/pacstall
+sudo mkdir -p /usr/share/man/man8/
+sudo mkdir -p /usr/share/bash-completion/completions
 
 rm -f $STGDIR/repo/pacstallrepo.txt > /dev/null
 touch $STGDIR/repo/pacstallrepo.txt
 echo 'https://raw.githubusercontent.com/pacstall/pacstall-programs/master' > $STGDIR/repo/pacstallrepo.txt
+
 fancy_message info "Pulling scripts from GitHub "
 for i in {add-repo.sh,search.sh,download.sh,install-local.sh,upgrade.sh,remove.sh,update.sh,query-info.sh}; do 
-	wget -q --show-progress -N https://raw.githubusercontent.com/pacstall/pacstall/master/misc/scripts/"$i" -P $STGDIR/scripts
-	chmod +x $STGDIR/scripts/*
-done &
+	wget -q --show-progress -N https://raw.githubusercontent.com/pacstall/pacstall/master/misc/scripts/"$i" -P $STGDIR/scripts &
```

Comment:
Missing `sudo`

Reply:
The installer's snippet is running with sudo already, like i said, I'll fix this whole mess of `install.sh` in another PR.

> sudo bash -c "$(curl -fsSL https://git.io/JsADh || wget -q https://git.io/JsADh -O -)"